README.md

```diff
@@ -1 +1,44 @@
-# firebase-queue-ui
+# Firebase Queue UI
+
+![](screenshot.png)
+
+### Setup
+
+Run the commands below to install dependencies
+
+```bash
+git clone git@github.com:isapp/firebase-queue-ui.git && cd firebase-queue-ui
+```
+
+```bash
+npm install && bower install
+```
+
+### Preview
+
+Run to preview and watch for changes
+
+```bash
+gulp serve
+```
+
+### Build
+
+Run to build for production
+
+```bash
+gulp
+```
+
+### Tests
+
+Run to execute the tests in the browser
+
+```bash
+gulp serve:test
```

Comment:
@viniciusalmeida since they're both working (cli and browser), I'll add them all to readme, ok? 👍